### ISSUE TITLE
make the "new map" button create a packaged map, and add a "new .bin" button to create one bin

### ISSUE THREAD

**Issue by microlith57** [CelestialCartographers/Loenn#13]
this would work into a larger plan for a more intuitive interface for managing campaigns, alt sides, and maps, and would also get rid of the problem of people being unwilling to package their maps / not packaging them correctly.

see https://discord.com/channels/403698615446536203/429775439423209472/934265980375408690

**Comment by MikosDrafle** [comment]
true, as the road of loenn becoming a new standard this will be useful for fellow mapmakers in the celeste community, and possibly a beginner option so mapmaking would be accessible for everyone
